hazelcast-spring/src/main/java/com/hazelcast/spring/cache/HazelcastCacheManager.java

```diff
@@ -110,7 +111,7 @@ public void setHazelcastInstance(HazelcastInstance hazelcastInstance) {
     }
 
     private void parseOptions(String options) {
-        if (null == options || options.trim().isEmpty()) {
+        if (StringUtil.isNullOrEmpty(options)) {
```

Comment:
```suggestion
        if (StringUtil.isNullOrEmptyAfterTrim(options)) {
```

Reply:
Not sure why we didn't apply this suggestion. Seems a correct suggestion to me.

Reply:
> Not sure why we didn't apply this suggestion. Seems a correct suggestion to me.

I pressed the wrong button :(